geode/dispatcher.py

```diff
@@ -220,12 +224,14 @@ def __init__(self, config=None):
                 )
             )
 
-        estimates = spatial.distance.cdist(
-            np.radians(origins),
-            np.radians(destinations)
-        ) * dist_metrics.R_EARTH
+        estimates = np.sqrt(np.square(
```

Comment:
seems to be no pair-wise form of distance functions that accepts arrays of lat longs.
this will accomplish same thing. euclidian distance, sqrt(a^2 + b^2))

Reply:
Not sure I understand. Can you explain differently maybe?

Reply:
regular dist function: ```dist([42, -88], [43, -87]) -> int```
scipy cdist: ```cdist([[42, -88], [43, -87]],  [[42, -88], [43, -87]]) -> 2x2 ints```
non-existent dist function: ```vdist([[42, -88], [43, -87]],  [[42, -88], [43, -87]]) -> 2x1 ints```

(pairwise distance seems to already have another meaning, so i'll just call it vdist)